### ISSUE TITLE
Generate Kickstart from an existing instance of FusionAuth

### ISSUE THREAD

**Issue by robotdan** [FusionAuth/fusionauth-issues#526]
### Problem
Kickstart is so awesome! But there are times when I want to take my dev or staging system as is and generate a kickstart so I can re-build the same configuration. 

### Solution
Add a feature to export a running system to a Kickstart file. This will likely exclude users, or allow for the selection of a few users intended to be admins or something like that. 

### Alternatives/workarounds
Build your own kickstart file. 

### Additional context
N/A

### How to vote
Please give us a thumbs up or thumbs down as a reaction to help us prioritize this feature. Feel free to comment if you have a particular need or comment on how this feature should work.

**Comment by mooreds** [comment]
Here's a capterra review asking for support for this feature: https://www.capterra.com/p/182987/FusionAuth/reviews/2365008/

**Comment by sanderkooger** [comment]
@dan, You are right, This would pretty much be Good method for a full export and import

**Comment by sanderkooger** [comment]
Ohhh I was just thinking.... How would Kubernetes react if you push all data into the kickstart.... Im guessing that might cause some deployment issues.

**Comment by mooreds** [comment]
@sanderkooger note that this would not export user password hashes. FusionAuth is pretty protective with those, but of course if you have database access, you could retrieve them and use the [Import API](https://fusionauth.io/docs/v1/tech/apis/users#import-users).
